receiver/signalfxreceiver/README.md

```diff
@@ -75,3 +67,7 @@ service:
       processors: [memory_limiter, batch]
       exporters: [signalfx]
 ```
+## Access token passthrough
```

Comment:
Since the option no longer exists, we can remove the README section here and just have the guidance in the changelog notes.